packages/drawer/src/mwc-drawer.ts

```diff
@@ -59,10 +68,9 @@ export class Drawer extends BaseElement {
         this._previousFocus = (this.getRootNode() as any as DocumentOrShadowRoot).activeElement as HTMLElement|null;
       },
       restoreFocus: () => {
+        document.$blockingElements.remove(this);
         const previousFocus = this._previousFocus && this._previousFocus.focus;
-        // Note, casting to avoid cumbersome runtime check.
-        const activeElement = (this.getRootNode() as any as DocumentOrShadowRoot).activeElement;
-        if (activeElement && this.mdcRoot.contains(activeElement) && previousFocus) {
+        if (previousFocus) {
```

Comment:
Removed `activeElement && this.mdcRoot.contains(activeElement)` from this condition, because at this point in the transition lifecycle, the drawer has already been closed. Determining whether the old activeElement was in the drawer or not is irrelevant.